app/event/send-return-event.js

```diff
@@ -0,0 +1,89 @@
+const raiseEvent = require('./raise-event')
```

Comment:
Is there too much going on in this file / does it need to be split out or made more modular.

Reply:
Yes, but as we're going to delete it when we remove V1 events, it's not worth the effort.  This is one of the many tech debt issues we resolve by switching to V2 events.